src/Neo/Network/P2P/Payloads/Signer.cs

```diff
@@ -66,6 +67,31 @@ public class Signer : IInteroperable, ISerializable
             /*AllowedGroups*/       (Scopes.HasFlag(WitnessScope.CustomGroups) ? AllowedGroups.GetVarSize() : 0) +
             /*Rules*/               (Scopes.HasFlag(WitnessScope.WitnessRules) ? Rules.GetVarSize() : 0);
 
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public bool Equals(Signer other)
```

Comment:
Is there a practical need for this API?

Reply:
those of being nulls will also be compared, i have checked,  null == null return true. Not sure if this can be of any problem, looks right to me.

Reply:
Is not bad to me also, == and != not required to me, but it's ok